pre-docs/plan_functions.md

```diff
@@ -460,7 +460,7 @@ run_task(String[1] $task_name, Boltlib::TargetSpec $targets, Optional[Hash[Strin
 
 * **task_name** `String[1]` The task to run.
 * **targets** `Boltlib::TargetSpec` A pattern identifying zero or more targets. See [`get_targets`](#get_targets) for accepted patterns.
-* **task_args** `Optional[Hash[String[1], Any]]` Arguments to the plan. Can also include additional options: '_catch_errors', '_run_as'.
+* **task_args** `Optional[Hash[String[1], Any]]` Arguments to the task. Can also include additional options: '_catch_errors', '_run_as'. Note: This is parameters for the task, not [configuration options](#bolt_configuration_options)
```

Comment:
I'm not sure this would have prevented the issue reported today. I think it may be more confusing to new users.